projects/plugins/jetpack/readme.md

```diff
@@ -0,0 +1,56 @@
+
+### Installation from the Jetpack Beta plugin
+
+If you cannot build the Jetpack plugin yourself, you can rely on [the Jetpack Beta plugin](https://github.com/Automattic/jetpack-beta/archive/master.zip) to install pre-built versions of Jetpack for you. To use the plugin, follow the instructions here:
+
+1. Go to Plugins > Add New in your dashboard.
+2. Install the Jetpack plugin.
+3. Go to Plugins > Add New > Upload Plugin.
+4. Upload and activate [this plugin](https://github.com/Automattic/jetpack-beta/archive/master.zip).
+5. Go to Jetpack > Beta and pick the version of Jetpack you would like to run on your site.
+
+## Contribute
+
+Thank you for thinking about contributing to Jetpack! See [the Jetpack Monorepo](https://github.com/Automattic/jetpack#contribute) for details on contributing.
+
+## Get Help
+
+Do you need help installing Jetpack, or do you have questions about one of the Jetpack modules? You can [search through our documentation here](https://jetpack.com/support/). If you don't find the answers you're looking for, you can [send us an email](https://jetpack.com/contact-support/) or [start a new thread in the WordPress.org support forums](https://wordpress.org/support/plugin/jetpack#new-post).
+
+## Security
+
+Need to report a security vulnerability? Go to [https://automattic.com/security/](https://automattic.com/security/) or directly to our security bug bounty site [https://hackerone.com/automattic](https://hackerone.com/automattic).
+
+## License
+
+Jetpack is licensed under [GNU General Public License v2 (or later)](./LICENSE.txt).
+
+## Team
+
+Jetpack is developed and maintained by various teams at Automattic. The Jetpack Crew responsible for coordinating everything is comprised of @anomiex, @bisko, @brbrr, @dereksmart, @fgiannar, @jeherve, @kbrown9, @kraftbj, @leogermani, @mdbitz, @sergeymitr and @zinigor.
+
+Contributions have been and continue to be made by dozens of other Automatticians, like:
+
+@aldavigdis, @allendav, @apeatling, @azaozz, @bazza, @beaulebens, @briancolinger, @cfinke, @daniloercoli, @chaosexanima, @ebinnion, @enejb, @eoigal, @georgestephanis, @gibrown, @gititon, @gravityrail, @jasmussen, @jblz, @jeffgolenski, @jeherve, @jessefriedman, @joanrho, @justinshreve, @keoshi, @koke, @kovshenin, @kraftbj, @lancewillett, @lezama, @martinremy, @mdawaffe, @MichaelArestad, @mtias, @mcsf, @mdawaffe, @nickmomrik, @obenland, @oskosk, @pento, @roccotripaldi, @stephdau, @Viper007Bond, @xyu and @yoavf.
+
+Our _awesome_ happiness engineers are @a8ck3n, @aheckler, @almoyen, @annezazuu, @bikedorkjon, @cena, @chaselivingston, @chickenn00dle, @coder-karen, @codestor4, @csonnek, @danjjohnson, @dericleeyy, @gaurav1984, @gsmumbo, @htdat, @jenhooks, @jeremypaavola, @joeboydston, @joendotcom, @lizthefair, @macmanx2, @madhattermattic, @mbhthompson, @mzakariya, @NujabesSoul, @ntpixels, @pmciano, @rachelsquirrel, @rcowles, @richardmtl, @snowmads, @stefmattana, and @tmmbecker.
```

Comment:
We should handle this in a follow-up PR, but a lot of these names aren't accurate anymore :-/